### PR TITLE
Update avie-dev.json

### PR DESCRIPTION
**Pull request by avie-dev**
add testimonials

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5008"></a>

### CODE REVIEW

**Review by mahfuztbt — APPROVED** [review]

**Comment by mahfuztbt** [comment]
LGTM

**Comment by avie-dev** [comment]
@mahfuztbt Thank you for the review🌷

**Review by accodes21 — APPROVED** [review]
All ready to merge💯

File: data/avie-dev.json [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {
   "name": "Avie Fukai",
   "type": "personal",
+  "displayStatsPublic": true,
```
Comment (eddiejaoude) [inline data/avie-dev.json:4]: This is no longer needed, as you can login to see your stats

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍 I left 1 small inline comment

File: data/avie-dev.json (hunk 1 above)
Comment (eddiejaoude) [reply data/avie-dev.json:4]:
```suggestion
```

Comment (avie-dev) [reply data/avie-dev.json:4]: Thank you Eddie!

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**